src/controllers/tileController.ts

```diff
@@ -0,0 +1,28 @@
+import { Request, Response } from "express";
+import { AppLocals } from "../types/app";
+
+export class TileController {
+    static getTile = async (req: Request, res: Response) => {
+        const { dataset, level, z, x, y } = req.params;
+        const { tileDatasets } = req.app.locals as AppLocals;
+
+        let tileData = null;
+        if (tileDatasets[dataset] && tileDatasets[dataset][level]) {
+            const db = tileDatasets[dataset][level];
```

Comment:
I like how these have been initialized on app start-up so we don't have to set up a new db connection on every request